README.md

```diff
@@ -36,24 +36,24 @@ The language is metaprogrammable and very extensible. You can easily build exten
 #### Hello World
 
 ```ruby
-fn main
+fn main()
   print "Hello World!"
```

Comment:
The language construct `print` doesn't exist anymore. It is a function, so, `print("Hello World!")`